src/content/security.txt

```diff
@@ -2,4 +2,4 @@ Contact: submariner-security@googlegroups.com
 Preferred-Languages: en, es, fr
 Canonical: https://submariner.io/security.txt
 Policy: https://submariner.io/development/security/
-Expires: Thu, 28 Oct 2021 00:00:00 -0000
+Expires: Thu, 12 Mar 2026 00:00:00 -0000
```

Comment:
Where did you get this date from?

Reply:
A year from the date I checked it. See the related docs update: https://github.com/submariner-io/submariner-website/pull/1240